.github/actions/godot-build-windows/action.yml

```diff
@@ -0,0 +1,48 @@
+name: Build Godot (Windows with CMD)
```

Comment:
Why we have this in the repo now?

Reply:
Because this runs in cmd, otherwise some parameters are not sent or replicated in bash. Mainly line 29, where it calls vcvarsall.bat, that is a script that adds some environment variables to the env needed for the build. Without it, one would need to run vcvarsall.bat in cmd and then copy-paste the env vars in the linux environment for the build to work.